src/Compilers/VisualBasic/Portable/Symbols/SynthesizedSymbols/SynthesizedIntrinsicOperatorSymbol.vb

```diff
@@ -31,12 +31,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
         Public Overrides ReadOnly Property IsCheckedBuiltin As Boolean
             Get
-                Select Case Me.Name
+                Select Case Name
```

Comment:
>Select Case Name [](http://example.com/codeflow?start=16&length=16)

It looks like there are no meaningful changes in this file. Consider reverting.

Reply:
This does not appear to be addressed.